tests/lib/processors/json.js

```diff
@@ -42,7 +42,7 @@ describe(function() {
       ignore: false,
       overrideConfig: {
         rules: {
-          'json-files/sort-package-json': [2]
+          'json-files/sort-package-json': ['error', { sortOrder: ['name', 'version', 'license'] }]
```

Comment:
I think these can/should be removed. These are testing the json parsing, and we don't need to exercise any sort-package-json options to do that.